microservices-demo/kustomize/base/recommendationservice.yaml

```diff
@@ -42,7 +42,7 @@ spec:
           privileged: false
           readOnlyRootFilesystem: true
         imagePullPolicy: Always
-        image: registry.digitalocean.com/microservices-demo/recommendationservice:1.0.0
+        image: <YOUR_DOCR_IMAGE>
```

Comment:
I don't think we need to replace everything with a placeholder, only the relevant part. Something like:

```
image: registry.digitalocean.com/<YOUR_DOCR_REPO_NAME_HERE>/recommendationservice:1.0.0
```